create_graphics.py

```diff
@@ -317,6 +360,12 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
         short_name=variable,
         )
 
+    try:
+        field.field
+    except errors.GribReadError:
+        print(f'Cannot find grib2 variable for {variable} at {level}. Skipping.')
+        return
+
```

Comment:
I followed up on this in master, and the code there is currently broken when you try to run all the maps for 0-6 hours. I think it would be best to skip fields we cannot find for that reason. As I mentioned in our meeting last week, that could lead to some tough debugging in the future. Are you both okay with that change, @Brian-Jamison and @cshartsough?

Reply:
Yes, it all looks good to me.  The discussion was helpful.

Reply:
I agree.  It's the best way to handle this.  Hopefully we can handle the tough debugging.